scanpipe/api/serializers.py

```diff
@@ -133,11 +133,11 @@ class Meta:
             "discovered_package_summary",
         ]
 
-    def get_codebase_resources_summary(self, project):
+    def get_codebase_resources_summary(self, project) -> dict:
         queryset = project.codebaseresources.all()
         return count_group_by(queryset, "status")
 
-    def get_discovered_package_summary(self, project):
+    def get_discovered_package_summary(self, project) -> dict:
```

Comment:
Not sure we want to mix and match type annotations. We should probably remove this.